velox/exec/WindowPartition.cpp

```diff
@@ -158,7 +158,9 @@ std::pair<vector_size_t, vector_size_t> WindowPartition::computePeerBuffers(
       peerStart = i;
       peerEnd = i;
       while (peerEnd <= lastPartitionRow) {
```

Comment:
lastPartitionRow is the last row of the partition when it fits in memory. What is the behavior now ? The last partition row can just be as much as the amount of data read.